documentation/modules/configuring/con-exposing-kafka-connect-api.adoc

```diff
@@ -0,0 +1,96 @@
+// This assembly is included in the following assemblies:
+//
+//assembly-config-kafka-connect.adoc
+
+[id='con-exposing-kafka-connect-api-{context}']
+
+= Exposing the Kafka Connect API
+
+[role="_abstract"]
+Use the Kafka Connect REST API as an alternative to using `KafkaConnector` resources to manage connectors.
+The Kafka Connect REST API is available as a service running on `_<connect_cluster_name>_-connect-api:8083`, where _<connect_cluster_name>_ is the name of your Kafka Connect cluster.
+The service is created when you create a Kafka Connect instance.
+
+NOTE: The `strimzi.io/use-connector-resources` annotation enables KafkaConnectors.
+If you applied the annotation to your `KafkaConnect` resource configuration, you need to remove it to use the Kafka Connect API.
+Otherwise, manual changes made directly using the Kafka Connect REST API are reverted by the Cluster Operator.
+
+You can add the connector configuration as a JSON object.
+
+.Example curl request to add connector configuration
+[source,curl,subs=attributes+]
+----
+curl -X POST \
+  http://my-connect-cluster-connect-api:8083/connectors \
+  -H 'Content-Type: application/json' \
+  -d '{ "name": "my-source-connector",
+    "config":
+    {
+      "class":"org.apache.kafka.connect.file.FileStreamSourceConnector",
```

Comment:
Together with @michalxo we found a typo in this example. The `class` should be `connector.class`. Personally, I would also put the `type` field there.